aperturedb/DataHelper.py

```diff
@@ -0,0 +1,159 @@
+
+from typing import Callable, Tuple
+import os
+from matplotlib.pyplot import plot_date
+import pandas as pd
+from aperturedb.ImageLoader import ImageGeneratorCSV, ImageLoader
+from kaggle.api.kaggle_api_extended import KaggleApi
+from kaggle.models.kaggle_models_extended import File
+
+from aperturedb import Connector
+import zipfile
+
+from aperturedb.ParallelLoader import ParallelLoader
+
+
+class DataHelper(object):
```

Comment:
I am not sure if I understand what is the purpose of this class. Is it not a general DataHelper, but rather a very specific set of methods to deal with Kaggle. 
We should have a general DataHelper class, and Kaggle being just 1 type of dataset we support.